src/plugins/settings/bool-setting/bool-setting.tsx

```diff
@@ -38,10 +38,12 @@ export class UIPBoolSetting extends UIPSetting {
   @memoize()
   protected get $inner(): HTMLElement {
     return (
-      <label>
-        {this.$field}
-        {this.label}
-      </label>
+      <>
```

Comment:
Do we need a fragment here?

Reply:
@ala-n probably not, will check it, thanks!